kong/constants.lua

```diff
@@ -133,4 +133,6 @@ return {
   },
   PROTOCOLS = protocols,
   PROTOCOLS_WITH_SUBSYSTEM = protocols_with_subsystem,
+  DEFAULT_ITERATION_SIZE = 1000,
+  DEFAULT_PAGE_SIZE = 100,
```

Comment:
I wouldn't be against `MAX_PAGE_SIZE` and `CLUSTER_EVENTS_PAGE_SIZE` as separate constants.

Reply:
these were added.